lib/sdam/server_description.js

```diff
@@ -38,6 +38,8 @@ const ISMASTER_FIELDS = [
   '$clusterTime'
 ];
 
+/** @typedef {'minWireVersion' | 'maxWireVersion' | 'maxBsonObjectSize' | 'maxMessageSizeBytes' | 'maxWriteBatchSize' | 'compression' | 'me' | 'hosts' | 'passives' | 'arbiters' | 'tags' | 'setName' | 'setVersion' | 'electionId' | 'primary' | 'logicalSessionTimeoutMinutes' | 'saslSupportedMechs' | '__nodejs_mock_server__' | '$clusterTime'} IsMasterProperties */
```

Comment:
Good catch will remove.
I was trying to do something clever to avoid all the default `this.x =` initializers I had to add, if it were .ts you can do this thing with an array of strings turn it into a union and then with an annotation tsc would know that these props exist on ServerDescription, without actually declaring them.